src/main/java/com/direwolf20/buildinggadgets/common/registry/OurItems.java

```diff
@@ -0,0 +1,132 @@
+    @ObjectHolder(ItemReference.PASTE_CONTAINER_T2)
+    public static ConstructionPasteContainer constructionPasteContainerT2;
+    @ObjectHolder(ItemReference.PASTE_CONTAINER_T3)
+    public static ConstructionPasteContainer constructionPasteContainerT3;
+    @ObjectHolder(ItemReference.PASTE_CONTAINER_CREATIVE)
+    public static ConstructionPasteContainerCreative creativeConstructionPasteContainer;
+
+    // Back to how the actual class works.
+    private static Set<Builder> itemBuilders = new HashSet<>();
+
+    /**
+     * setup us required to be loaded before the registry event
+     * happens.
+     */
+    public static void setup() {
+        // Looks complicated but it's not. We're just building a list of Builders that will, when
+        // applied will construct the Item.Properties Builder for us allowing us to do a lot less work.
+        itemBuilders.addAll(new HashSet<Builder>() {{
+            add(new Builder(ItemReference.GADGET_BUILDING_RL).setBuilder(nonStackableItemProperties().maxDamage(1)).setFactory(GadgetBuilding::new));
+            add(new Builder(ItemReference.GADGET_EXCHANGING_RL).setBuilder(nonStackableItemProperties().maxDamage(1)).setFactory(GadgetExchanger::new));
+            add(new Builder(ItemReference.GADGET_COPY_PASTE_RL).setBuilder(nonStackableItemProperties().maxDamage(1)).setFactory(GadgetCopyPaste::new));
+            add(new Builder(ItemReference.GADGET_DESTRUCTION_RL).setBuilder(nonStackableItemProperties().maxDamage(1)).setFactory(GadgetDestruction::new));
+            add(new Builder(ItemReference.PASTE_CONTAINER_T1_RL).setBuilder(nonStackableItemProperties()).setFactory(RegularPasteContainerTypes.T1::create));
+            add(new Builder(ItemReference.PASTE_CONTAINER_T2_RL).setBuilder(nonStackableItemProperties()).setFactory(RegularPasteContainerTypes.T2::create));
+            add(new Builder(ItemReference.PASTE_CONTAINER_T3_RL).setBuilder(nonStackableItemProperties()).setFactory(RegularPasteContainerTypes.T3::create));
+            add(new Builder(ItemReference.PASTE_CONTAINER_CREATIVE_RL).setBuilder(nonStackableItemProperties()).setFactory(ConstructionPasteContainerCreative::new));
+            add(new Builder(ItemReference.CONSTRUCTION_PASTE_RL).setBuilder(itemProperties()).setFactory(ConstructionPaste::new));
+            add(new Builder(ItemReference.CONSTRUCTION_CHUNK_DENSE_RL).setBuilder(itemProperties()).setFactory(Item::new));
+            add(new Builder(ItemReference.TEMPLATE_RL).setBuilder(itemProperties()).setFactory(Template::new));
+        }});
+    }
+
+    /**
+     * Register the items with the forge registry by looping over
+     * our builders and calling their register method.
+     *
+     * @param event the forge registry event
+     */
+    @SubscribeEvent
+    public static void registerItems(RegistryEvent.Register<Item> event) {
+        itemBuilders.forEach(e -> event.getRegistry().register(
+                e.register()
+        ));
```

Comment:
Otherwise it's a small memory leak... You could also free in on load finished, which is what I used to do. 
```suggestion
        ));
        itemBuilders.clear();
```